### DISCUSSION TITLE
Xcode 14.0.1 and Xcode 14.1 Beta 2

### DISCUSSION THREAD

**Discussion by mikhail-tokarev** [codemagic-ci-cd/codemagic-docs#959] [Announcements]
**Xcode 14.0** and **Xcode 14.1 Beta 1** are now available on our macOS build machines.

You can use these versions to run the build if you specify `14.0` or `14.1` as the Xcode version in your build settings.

⚠️ On September 26, we will point `latest` to `14.0` and `edge` to `14.1`.

See the build machine specs for information about the preinstalled software and available runtimes:
- [macOS M1](https://docs.codemagic.io/specs/versions-macos-m1-xcode-13-3/)
- [macOS (Intel)](https://docs.codemagic.io/specs/versions-macos-xcode-13-3/)

**Comment by huynguyennang-fruitstreet** [comment]


**Reply by mikhail-tokarev** [reply]
hey @huynguyennang-fruitstreet 
for issue reports start a new discussion please

**Comment by mikhail-tokarev** [comment]
**UPDATE:** `latest` points to Xcode 14.0.1 and `edge` now points to Xcode 14.1 Beta 2 versions
